src/trade-params/TradeParams.js

```diff
@@ -56,12 +70,12 @@ export default class TradeParams extends Component {
         contract: PropTypes.object,
         compact: PropTypes.bool,
         disabled: PropTypes.bool,
-        error: PropTypes.string,
+        errors: PropTypes.object,
```

Comment:
Exactly, which is why I propose to defer immutableToJS, to retain the equality, sometime it's just makes more sense >.<